apps/client/src/components/Location/LocationFinder.js

```diff
@@ -85,7 +85,7 @@ const LocationFinder = (props) => {
           name="postalCode"
           label="Postcode"
           defaultValue={postalCode}
-          error={postalCodeError}
+          error={postalCodeError || (notFoundAddress && !graphqlError)}
```

Comment:
This was very confusing for me and took a long while to figure out.

I assumed `error` accepts a string, but it doesn't. In our case postalCodeError is a string but used as a bool. Please create a new var `hasPostalCodeError` and double negate `!!` it, or refactor in another way.
Also, what does `(notFoundAddress && !graphqlError)` mean. Address not found in Amsterdam?
Also, why don't we use the `errorMessage` prop on `<TextField />` instead of `<ErrorMessage />`?
This took me a while to figure out. Can you refactor a bit, add some comments?
And I think all this was extra confusing because it's on this upgrade-branch, but it's completely not related right?

Reply:
How can I test this?

Reply:
true! will explain later, for now I will remove it. 

btw: `errorMessage` prop in TextField is deprecated: use `ErrorMessage` component instead.